client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.tsx

```diff
@@ -168,9 +168,13 @@ export const NumericTextInput: FC<NumericTextInputProps> = React.forwardRef(
           onChange(newNum);
         }}
         onBlur={() => {
+          const defaultVal = min === 0 ? undefined : min;
```

Comment:
If a min value is defined, and the input is undefined, we should set the value to that min on blur

Reply:
Hmm, I don't think this is the behaviour we want. If there's a minimum, you should still be able to remove your input.

Also -- this causes a problem with negative values. The default "allowNegative" sets a minimum of `-Infinity` (or largest integer), so if the `allowNegative` prop is set, then when you remove your input and leave the input, it jumps to -999,999,999! 

(We should probably add an additional test for this case)

Reply:
If we need to enforce a compulsory minimum, this should be the responsibility of the calling component, IMHO.